packages/itwinui-css/src/style/global.scss

```diff
@@ -65,3 +65,9 @@ html {
   // Scrollbar
   @include iui-scrollbar;
 }
+
+[class*='iui-'],
```

Comment:
ok, lets update this selector to exclude body and toast images should be reverted then
```
[class*=iui-]:where(:not(.iui-body)),
[class*=iui-]:where(:not(.iui-body))::before,
[class*=iui-]:where(:not(.iui-body))::after{
  box-sizing:border-box;
}
```

This should not increase specificity and also we will not influence users app in any way

Reply:
Did it this way and reverted toast tests